src/components/select/multi-select/multi-select-test.stories.tsx

```diff
@@ -537,3 +537,34 @@ export const MultiSelectWithDisabledOption = () => {
     </MultiSelect>
   );
 };
+
+export const SingleOption = () => {
+  const MAX_SELECTIONS_ALLOWED = 2;
```

Comment:
**nitpick(non-blocking):** I think this could be removed from the story since there is only one option